integreat_cms/cms/templates/analytics/__page_hix_row.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+{% load rules %}
+{% load content_filters %}
+{% load page_filters %}
+{% with page_translation.page.id as page_id %}
+    {% with page_translation.hix_score as hix %}
+        <tr class="hover:bg-gray-100 {% if hix < hix_threshold %}text-red-500{% endif %}">
```

Comment:
```suggestion
        <tr class="{% if hix < hix_threshold %} bg-red-100 hover:bg-red-200 text-red-500 hover:text-red-600 border-l-4 border-red-500 {% else %} hover:bg-gray-100 border-l-4 border-white hover:border-gray-400 {% endif %}">
```
I liked the solution with the borders, what do you think about this styling?